canonical-websites/maas.io
# Update to tour hero style and copy.

## Done

Updated the styling of the hero on the tour page - changed the background image and some positioning tweaks.

Updated the hero copy.

## QA

 - Go to /tour
 - Visually check the hero at all viewport sizes
 - Check the copy against the [copydoc](https://docs.google.com/document/d/1KcYwNK0pJyiDVEDq4vTpv6BX_rpSDnxUxtAvAnsRKyw/edit#heading=h.d2yomvm8pmy4)
